tools/cachetop.py

```diff
@@ -157,7 +157,7 @@ def handle_loop(stdscr, args):
         u32 uid = bpf_get_current_uid_gid();
 
         key.ip = PT_REGS_IP(ctx);
-        key.pid = pid & 0xFFFFFFFF;
+        key.pid = pid >> 32;
         key.uid = uid & 0xFFFFFFFF;
```

Comment:
Since you are on this, could you remove `& 0xFFFFFFF` for key.uid. uid is u32, masking is unnecessary.